### ISSUE TITLE
[registration] "How to find the reason why the ndt FitnessScore is too large"

### ISSUE THREAD

**Issue by daohu527** [PointCloudLibrary/pcl#5134]
**Is your feature request related to a problem? Please describe.**
We often use NDT for point cloud registration, and it also gives a `FitnessScore`. If the `FitnessScore` is small, we are all happy, but when it is large, how can we find the problem?

**Context**

Especially when the 2 models are about the same, how do we know where is a bad match and where is a good match, instead of giving an average about `all voxels`.

**Expected behavior**

Can we visualize different voxel errors, like a depth map to show the difference?

**Current Behavior**

Just a `FitnessScore` and the output point cloud

**Describe the solution you'd like**

Display the error of each voxel, and sort

**Comment by mvieth** [comment]
I found this open pull request: #2895 Is that what you had in mind?

**Comment by daohu527** [comment]
Yes, Thank you for your reply! I saw this pr, the idea is very good, but I don't agree with its design.

**Comment by mvieth** [comment]
What kind of "new interface" are you thinking of? If you could suggest some function signatures, that would be great

**Comment by daohu527** [comment]
Thank you for your reply! I will provide some reference interfaces, and then you can see if it is suitable : )
